openwisp_monitoring/device/api/views.py

```diff
@@ -200,6 +201,22 @@ def _write(self, request, pk):
                 metric.write(client['mac'])
             if created:
                 self._create_clients_graph(metric)
+        if 'resources' in data:
+            resources = {}
+            if 'load' in data['resources']:
+                resources['load'] = 100 * data['resources']['load'][1]
+            if 'disk_usage' in data['resources']:
+                used_space, total_size = 0, 0
+                for disk in data['resources']['disk_usage']:
+                    used_space += disk['used_space']
+                    total_size += disk['size']
+                resources['disk_usage'] = 100 * used_space / total_size
+            if 'memory' in data['resources']:
+                memory = data['resources']['memory']
+                used = 1 - (memory['free'] - memory['buffered']) / memory['total']
+                resources['memory'] = 100 * used
```

Comment:
Same here, save all data, and then we can calculate averages and so on from the influxdb query